src/Components/MyProfile/MyProfile.js

```diff
@@ -0,0 +1,46 @@
+import { useSelector } from 'react-redux';
+import RocketsReserved from '../Rockets/RocketsReserved';
+import styles from './myProfile.module.css';
+
+const MyProfile = () => {
+  const allDragons = useSelector((state) => state.dragonReducer);
+  const missionsState = useSelector((state) => state.missionsReducer.missions);
+  const joinedMissions = missionsState.filter(
+    (missions) => missions.reserved === true,
+  );
+
+  const reservedDragons = allDragons.filter((dragons) => !dragons.reserved !== true);
```

Comment:
- [ ] [OPTIONAL] you can add a message that states there are no available reserved dragons/missions when they are no reservered dragons/missions. that way it is clear to the user that this is the case.

Reply:
Thanks, man!  we'd put that in mind for other cases like that